docs/Guide.Jest.md

```diff
@@ -44,16 +43,14 @@ beforeEach(async () => {
 Add this part to your `package.json`:
 ```json
 "jest": {
-  "setupTestFrameworkScriptFile": "<rootDir>/jest/setup.js"
+  "setupTestFrameworkScriptFile": "./e2e/init.js"
 },
 "scripts": {
-    "test:e2e": "jest __e2e__ --setupTestFrameworkScriptFile=./jest/setup/e2e-tests.js --runInBand",
+    "test:e2e": "detox test -r jest",
     "test:e2e:build": "detox build"
 }
 ```
 
-We need the `--runInBand` as detox doesn't support parallelism yet.
```

Comment:
Will running without the flag run tests serially?

Reply:
I added the flag [here](https://github.com/wix/detox/pull/335/files#diff-7ba9426c8fa0338c47632e8dbf8c11f3R42) so if you use `detox test -r jest` it is always there

Reply:
What is stopping detox from parallelism?

Reply:
a few things, mainly controlling multiple simulators. It's in our roadmap, shouldn't be too hard.